update flag image version number (fixes #73)

thedavidthomas opened the pull request:
swapping out version number on flag image sprite 

before:
<img width="429" alt="Screenshot 2023-08-08 at 3 07 55 pm" src="https://github.com/Log1x/acf-phone-number/assets/2648750/fb02a81e-e0ef-49c3-bf77-a54a878fa69e">

after:
<img width="532" alt="Screenshot 2023-08-08 at 3 07 28 pm" src="https://github.com/Log1x/acf-phone-number/assets/2648750/2229805a-820c-4f15-abce-3847831fe7e5">

Log1x commented:
thanks! can you remove the extra yarn stuff? can also probably do `yarn upgrade --latest` and swap the flags to use 18.2.1 instead.

thedavidthomas commented:
sorry am new to yarn (but noticed you had yarn.lock so thought should use it), volta my node tool decided to install yarn 4.0 which added that extra stuff.

switched to yarn 3.6.1 now and removed extra stuff that yarn 4.0 added  (hopefully this is good now) 
upgraded packages and flags to 18.2.1
